index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="navbar" class="main-nav">
+        <a href="#">Home</a>
+        <a id="profile-link" href="https://github.com/" target="_blank">Git Hub</a>
+    </nav>
+    <section id="welcome-section">
+        <h1>Welcome to our webpage</h1>
+        
+    <!-- <img src="https://thumbs.dreamstime.com/b/projects-concept-black-chalkboard-d-rendering-handwritten-top-view-office-desk-lot-business-office-supplies-79906734.jpg" alt="an image"> -->
```

Comment:
I think it would be better to remove unused old code comments like this one to make your code look cleaner 👍

Reply:
- [ ] comment offered as a suggestion with the reason for change request 👏🏾